docker/conda/environments/cuda11.5_dev.yml

```diff
@@ -32,9 +32,9 @@ dependencies:
     - cmake=3.24
     - configargparse=1.5
     - cuda-nvml-dev=11.5
-    - cuda-python<=11.5.0 # Remove when Issue #251 is closed
+    - cuda-python>=11.6 # Remove when Issue #251 is closed
```

Comment:
Does this PR fix issue 251?

Reply:
Looks like they added an exclusion so its not possible to install greater than 11.7.0